test/jdk/javax/swing/text/html/StyleSheet/TestWrongCSSFontSize.java

```diff
@@ -134,7 +146,8 @@ private static void captureImage(Component comp, String path) {
     }
 
     public static void main(String[] args) throws Throwable {
-        TestWrongCSSFontSize test = new TestWrongCSSFontSize();
+        TestWrongCSSFontSize test = new TestWrongCSSFontSize(
```

Comment:
A minor nit...
Actually in all ImageIO.write() we normally pass the file with extension like image.png but here if the argument passed to testcase does not give .png extension, it just stores as that name, which is kind of irritation. I think we can hardcode the filename in call to ImageIO.write to CSSimage.png or something similar, instead of getting the path in main.

Reply:
This is a side effect of my change: one parameter is always passed to the test and now the test always save an image; the test should not save the image by default. To enable saving the image, a second parameter needs to be passed. Hardcoding the name of the image, perhaps based on the first parameter `w3cUnits` seems reasonable.

Reply:
I guess instead of depending on passed parameter, it will be better to capture the image only on failure.

Reply:
> I guess instead of depending on passed parameter, it will be better to capture the image only on failure.

That's an option. However, I don't think the image adds much value here.

Reply:
It might be during failure :-) [which is why it is captured during failure for many other regression tests]

Reply:
Yeah, sure. Likely, this kind of issue will be reproducible in other environments if it fails.
My initial idea during #1759 was to show a frame which would be useful during debugging.

I hope it will never fail again. :)

Anyway, I'm okay with whatever solution @stanio implements. The easiest one, however, is to use the second parameter to save the image. Sometimes it's useful to have the image captured even when the test does not fail.

Reply:
I had thought it would be most useful to have it save an image capture on failure during an automated run, but I was not sure what file path is possible/appropriate to use – would using the current working directory be o.k.?  I may fix the image capture file then, and possibly use a program argument just to force saving a successful run capture – does it sound o.k.?

Reply:
Current directory is fine. The test harness sets it before running the test and then it captures the files there.
When running the test manually, the current directory is also the most intuitive choice.

Reply:
@prsadhuk, see if the last change to `TestWrongCSSFontSize` here satisfies your expectation.